.github/workflows/server-function.yml

```diff
@@ -1,49 +0,0 @@
-name: ci
-
-on:
-  pull_request:
-    branches: [main]
```

Comment:
why is this file removed?

Reply:
renamed so it's not only about server-functions only. Git tracked as a removal/addition because I changed too much inside to run unit and e2e from the same built artifact instead of building things only once.